internal/rbac/migrate.go

```diff
@@ -0,0 +1,41 @@
+package rbac
+
+import (
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"github.com/ucl-arc-tre/portal/internal/graceful"
+	"github.com/ucl-arc-tre/portal/internal/types"
+	"gorm.io/gorm"
+)
+
+func runMigrations() {
+	db := graceful.NewDB()
+	migrateStudyAdmins(db)
+	log.Debug().Msg("Migrated RBAC")
+	// todo - create issue to remove rbac migrations
+}
+
+func migrateStudyAdmins(db *gorm.DB) {
+	studyIds := []uuid.UUID{}
+	if err := db.Model(&types.Study{}).Select("id").Find(&studyIds).Error; err != nil {
+		panic(err)
+	}
```

Comment:
Do we need both the rbac and the `study_admins` table, or is it possible to get away with just the rbac?